### DISCUSSION TITLE
Qual a sua carga horária semanal?

### DISCUSSION THREAD

**Discussion by marcosalberto** [frontendbr/forum#1218] [Discussão]
Olá pessoal, tudo bom?

Quero entender como está a carga horária semanal das empresas pelo Brasil, principalmente para quem trabalha com CLT.

Muito obrigado.

**Comment by felipemfp** [comment]
Tudo jóia!

Na minha cidade (Natal/RN), nunca vi nada além de 40h/sem com CLT.

**Comment by laforetmoreno** [comment]
Opa, blz ?

Na minha carteira esta 40h/semana, mas raramente rola isso. (Rio de Janeiro)

**Comment by caioincau** [comment]
40h/semana e tento manter isso, quase sempre consigo.

**Comment by tunnes** [comment]
Tudo certo e você?

Bom trabalhando PJ também tento manter por volta de ~40h, lendo agora sua pergunta fiquei curioso sobre a carga horária na Alemanha em especial em Berlim, me recordo vagamente de ter lido algo sobre 28h alguém trampa por lá para confirmar? 🤔

**Comment by didiraja** [comment]
Depende da empresa, se voce trabalhar numa agência de publicidade. É comum fazer hora exra com frequencia.

**Comment by felipefialho** [comment]
176h/mensal 

Mas depende muito da regulação da empresa que você trabalha.

**Comment by kvnol** [comment]
44hrs/CLT :cry:

**Comment by caueops** [comment]
44h/semana

Sobre o modelo de contrato CLT, é direito nosso não ultrapassar a carga de 8h/dia e 44h/semana e considero difícil de você assinar um contrato CLT diferente dessa carga horária aqui no Brasil.

O que acontece, que consequentemente aumenta a carga horária na minha opinião, é a própria cultura da empresa em que você está.

**Comment by caioincau** [comment]
Exato, vários lugares fazem pressão, já vi um caso onde todos batiam palma para o primeiro a sair da empresa(mesmo esse cara tendo passado da hora normal), como forma de "coação" .

**Comment by victornery** [comment]
Trabalho numa Agência de Publicidade aqui em Fortaleza e aqui é 44hrs/semanal CLT.

**Comment by renanbastos93** [comment]
44 hrs semanais pelo contrato CLT em Porto Alegre, RS

**Comment by ingridrauany** [comment]
40hrs semanais - CLT - Montes Claros/MG

**Comment by kavalcante** [comment]
Atualmente, em média 60 horas, sendo em torno de 20-40 com o editor aberto codando propriamente dito (Wakatime).

O resto é planejando, conversando, tentando chegar a alguma coisa.

**Comment by rodrigooler** [comment]
Depende muito, mas atualmente em torno de 50 horas semanais

**Comment by thulioph** [comment]
35hrs semanais - CLT - Recife/PE

**Comment by praganavictor** [comment]
Eu faço 50 horas semanais (trabalho em uma produtora) PJ - Sao Paulo - SP

**Comment by vhfmag** [comment]
Na carteira está como 40 horas semanais, mas a carga horária prática frequentemente passa disso

**Comment by marcosalberto** [comment]
Obrigado pela contribuição galera, o intuito era entender se a maioria trabalha 44 ou 40 horas na CLT. Mesmo com poucas respostas aparentemente não há um padrão, está bem dividido.

**Comment by felipefialho** [comment]
Nesse [Survey](https://survey.frontendbr.com.br/) que lancei semana passada da pra ter uma boa noção disso.
